week-1-ts-ecosystem/w01e01-typescript-basics/src/types.ts

```diff
@@ -0,0 +1,11 @@
+interface Address {
+  street: string;
+  no: number;
+}
+
+export interface User {
+  name: string;
+  age: number;
+  email: string;
+  address?: Address;
+}
```

Comment:
To `types.ts` to też dość "uogólniona nazwa".

Co powiesz na `user.model.ts`

podoba mi się że zauważasz tutaj 2 typy i rozdzielasz `Address` i `User`
oraz na razie nie udostępniasz `Address` bo nie ma takiej konieczności - cool.